py/ai/h2o/sparkling/ml/algos/H2OAutoML.py

```diff
@@ -16,52 +16,26 @@
 #
 
 from pyspark import keyword_only
-from pyspark.ml.util import JavaMLWritable, JavaMLReadable
-from pyspark.ml.wrapper import JavaEstimator
+from pyspark.sql.dataframe import DataFrame
 
 from ai.h2o.sparkling import Initializer
-from ai.h2o.sparkling.ml.algos.H2OAutoMLExtensions import H2OAutoMLExtensions
-from ai.h2o.sparkling.ml.models import H2OMOJOModel
+from ai.h2o.sparkling.ml.algos.H2OAlgoBase import H2OAlgoBase
 from ai.h2o.sparkling.ml.params import H2OAutoMLParams
-from ai.h2o.sparkling.sparkSpecifics import get_input_kwargs
 
 
-class H2OAutoML(H2OAutoMLParams, H2OAutoMLExtensions, JavaEstimator, JavaMLReadable, JavaMLWritable):
+class H2OAutoML(H2OAutoMLParams, H2OAlgoBase):
 
     @keyword_only
-    def __init__(self, featuresCols=[], labelCol="label", allStringColumnsToCategorical=True, columnsToCategorical=[], splitRatio=1.0, foldCol=None,
-                 weightCol=None, ignoredCols=[], includeAlgos=None, excludeAlgos=None, projectName=None, maxRuntimeSecs=3600.0, stoppingRounds=3,
-                 stoppingTolerance=0.001, stoppingMetric="AUTO", nfolds=5, convertUnknownCategoricalLevelsToNa=True, seed=-1,
-                 sortMetric="AUTO", balanceClasses=False, classSamplingFactors=None, maxAfterBalanceSize=5.0,
-                 keepCrossValidationPredictions=True, keepCrossValidationModels=True, maxModels=0,
-                 predictionCol="prediction", detailedPredictionCol="detailed_prediction", withDetailedPredictionCol=False,
-                 convertInvalidNumbersToNa=False, **deprecatedArgs):
+    def __init__(self, **kwargs):
```

Comment:
The same concern as [here](https://github.com/h2oai/sparkling-water/pull/1426/files#r312550016).

Reply:
Was thinking about this and still wasn't sure what was better win. 

We can still generate the code, but have the arguments mentioned just once in the ``__init__ `` method. The 2 other just duplicates already specified information

Reply:
I put the arguments back into init method, we can iterate later, but we can probably generate it later